src/Toolkit.Forms/Bookmarks/BookmarksView.xaml.cs

```diff
@@ -105,7 +118,7 @@ public GeoView GeoView
         /// Identifies the <see cref="ItemTemplate" /> bindable property.
         /// </summary>
         public static readonly BindableProperty ItemTemplateProperty =
-            BindableProperty.Create(nameof(ItemTemplate), typeof(DataTemplate), typeof(BookmarksView), new DataTemplate(typeof(BookmarkItemTemplate)), BindingMode.OneWay, null, propertyChanged: ItemTemplateChanged);
+            BindableProperty.Create(nameof(ItemTemplate), typeof(DataTemplate), typeof(BookmarksView), DefaultDataTemplate, BindingMode.OneWay, null, propertyChanged: ItemTemplateChanged);
```

Comment:
Don't set mutable reference types as default values in the static properties. Instead assign in the constructor and make it null here.

Reply:
I was following the pattern from Legend. How is that different from what's being done there? 
https://github.com/Esri/arcgis-toolkit-dotnet/blob/007cea730683bd50729a03085073703884fa22a4/src/Toolkit.Forms/Legend/Legend.cs#L221

Reply:
Oops! It's not, and that's a problem too :)

Reply:
Brainfarting today. Move along. Nothing to see here...